src/OrchardCore/OrchardCore.Abstractions/Localization/CultureScope.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Globalization;
+
+namespace OrchardCore.Localization
+{
+    public sealed class CultureScope : IDisposable
+    {
+        private CultureScope(CultureInfo culture)
+        {
+            Culture = culture;
+            CultureInfo.CurrentUICulture = culture;
+        }
+
+        public CultureInfo Culture { get; private set; }
```

Comment:
I would not use a property, just a private field

Reply:
Just a second you will see where could be used ;)